web/frontend/src/app.vue

```diff
@@ -617,16 +605,10 @@ const waitForClientAndStart = async () => {
   }
 };
 
-const handleSelectCamera = async (event: string) => {
-  if (event === '') {
-    await Promise.all(activePreviews.map((preview) => viewCamera(preview, false)));
-    activePreviews = [];
-    return;
-  }
+let baseCameraState = new Map<string, boolean>();
```

Comment:
Can we declare this up near the top of the file before it's used in any function?